src/octoprint/static/js/app/viewmodels/gcode.js

```diff
@@ -566,6 +553,38 @@ $(function() {
         self.onTabChange = function(current, previous) {
             self.tabActive = current == "#gcode";
         };
+        
+        self.incrementLayer = function(value){
```

Comment:
That's a bit of a misleading name - it doesn't actually increment but rather changes the layer. What about `changeLayer` instead?